src/components/file-upload/index.md

```diff
@@ -35,16 +35,18 @@ Help users select and upload a file.
 
 ## When to use this component
 
-You should only ask users to upload something if it’s critical to the delivery of your&nbsp;service.
+You should only ask users to upload something if it’s critical to the delivery of your service.
 
-We improved the component in February 2025, you will need to [enable the improved File upload component as a new feature](#using-the-improved-file-upload-component).
+We improved the component in March 2025, so you’ll need to enable the improved File upload component as a new feature.
```

Comment:
@seaemsi Looks like we've removed the link to the section further down the page, just checking that it's intentional (for ex. because that section is not too far after) 😊